### PR TITLE
Updated links to XPAAS images per BZ

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1557188

### CODE REVIEW

**Comment by mburke5678** [comment]
Based on the compatibility matrix, all the XPAAS image versions are the same for 3.7 and 3.9 only.
JDG 6.5 iss required for 3.2-3.5.
Fuse 6.2 is required for 3.2 and 3.3.
Fuse 6.3 is required for 3.4+

Will change 3.2-3.6 branches as needed in separate PRs.

File: install_config/imagestreams_templates.adoc [hunk 1]
```diff
@@ -104,12 +104,12 @@ ifdef::openshift-dedicated,openshift-enterprise[]
...
-- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-fuse-integration-services-for-openshift/[Red Hat JBoss Fuse Integration Services]
+a|- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_a-mq/6.3/html-single/red_hat_jboss_a-mq_for_openshift/[Red Hat JBoss A-MQ]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_bpm_suite/6.4/html-single/red_hat_jboss_bpm_suite_intelligent_process_server_for_openshift/[Red Hat JBoss BPM Suite Intelligent Process Server]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_brms/6.4/html-single/red_hat_jboss_brms_realtime_decision_server_for_openshift/[Red Hat JBoss BRMS Decision Server]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_data_grid/7.1/html-single/data_grid_for_openshift/[Red Hat JBoss Data Grid]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/red_hat_jboss_enterprise_application_platform_for_openshift//[Red Hat JBoss EAP]
```
Comment (wzheng1) [inline install_config/imagestreams_templates.adoc]: I can see there are two "/" in the end of the link, it seems one of them needs to be removed, right?

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. Minor change.

**Comment by ahardin-rh** [comment]
LGTM

**Pull request merged**